### PR TITLE
Introduced performance test for most used content bundle

### PR DESCRIPTION
**Pull request by mtodor**
**Description**
This PR introduces a test for performance testing that will discover and dynamically fill fields related to a specific content bundle.

This PR requires end-point information provided by new controller introduced in the following PR:
https://github.com/thunder/thunder_performance_measurement/pull/1

**Info:**
- [x] All coding styles are fulfilled. ([How to check for cs issues?](https://github.com/BurdaMagazinOrg/thunder-dev-tools/blob/master/README.md#code-style-guidelines))
- [x] All tests are running locally. ([How to run the test?](https://github.com/thunder/thunder-distribution/blob/8.x-3.x/docs/development.md#how-to-run-the-tests))
- [x] Code is covered with well-balanced amount of inline comments.

### CODE REVIEW

File: tests/src/Nightwatch/Tests/CreateMostUsedContentBundle.js [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+        // Get Site information first!!!
+        const { data } = JSON.parse(body);
+
+        browser._site_info = data;
+
+        done();
+      }
+    );
+  },
+  createMostUsedContentBundle(browser) {
```
Comment (chrfritsch) [inline tests/src/Nightwatch/Tests/CreateMostUsedContentBundle.js]: The name is a bit misleading. This function creates content with the most used bundle

File: tests/src/Nightwatch/Tests/CreateMostUsedContentBundle.js [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+    const { bundle, required_fields: requiredFields } = browser._site_info;
+
+    browser
+      .resizeWindow(1024, 1024)
+      .drupalRelativeURL("/user/login")
```
Comment (chrfritsch) [inline tests/src/Nightwatch/Tests/CreateMostUsedContentBundle.js]: I think there is no need to visit the login page manually. drupalLogin will do that for us

**Review by chrfritsch — CHANGES_REQUESTED** [review]

File: docs/development.md [hunk 3]
```diff
@@ -106,6 +106,14 @@ For cases like this you can set environment variable `MINK_DRIVER_ARGS_WEBDRIVER
 
 That information will be picked up by testing classes and used for selenium endpoint.
 
+#### How to run the NightwatchJS performance tests
+
+1. You need to install [Yarn](https://yarnpkg.com). Please check installation documentation for it.
```
Comment (chrfritsch) [inline docs/development.md:111]: I think we can replace 1. by "Follow the 'Running Nightwatch tests' docs in core/tests/README.md". Then we can also remove 4.

Comment (mtodor) [reply docs/development.md:111]: I would keep it simple and straight forward.

**Review by chrfritsch — APPROVED** [review]

**Pull request merged**